cmd/main.go

```diff
@@ -58,15 +62,17 @@ var splitCmd = &cli.Command{
 				}
 				return err
 			}
-			path := fmt.Sprintf("%s/%s-%d.car", dir, name, i)
-			fmt.Printf("Writing CAR chunk to %s\n", path)
-			fi, err := os.Create(path)
+			writePath := fmt.Sprintf("%s/%s-%d.car", dir, name, i)
+			fmt.Printf("Writing CAR chunk to %s\n", writePath)
+			fi, err := os.Create(writePath)
 			if err != nil {
 				return err
 			}
 			br := bufio.NewReader(r)
-			br.WriteTo(fi)
-			fi.Close()
```

Comment:
We should still close the file...

Reply:
This is where my go noobness comes to life, the ide tripped me up.